WalletWasabi.Tests/UnitTests/Crypto/ZeroKnowledge/TranscriptTests.cs

```diff
@@ -111,6 +111,7 @@ public void SyntheticNoncesSecretDependence()
 			var mrb = new MockRandom();
 			mra.GetBytesResults.Add(new byte[32]);
```

Comment:
my mistake was a typo or copypaste error in the first call (uses `mra` not `mrb`) so it should be just 2 calls, gonna push a fix